sysprep/instance_setup.ps1

```diff
@@ -220,6 +220,9 @@ if ($specialize) {
     Write-LogError
   }
 
+  Write-Log 'Enable google_osconfig_agent during the specialize configuration pass.'
```

Comment:
I'd suggest moving this to above line 208 `if ($specialize) {`, given Andrew's comment of putting it first in the script in case anything goes wrong. I also think we're want this to run regardless of the $specialize value.

Reply:
Thank you. I will update the code logic in the update to this PR.

Reply:
Integration test created and merged in a separate internal CL. Will soon send a revision to this PR. Thank you.